### DISCUSSION TITLE
Adding image alt text

### DISCUSSION THREAD

**Discussion by leo-petrucci** [Milkdown/milkdown#739] [Q&A] [answered]
Is there any way to add alt text to an image outside of directly editing the markdown?

If not, how easy is it to override the way the wysiwyg editor displays image components to add this functionality?

**Comment by Saul-Mirone** [comment, marked as answer]
There's no way out of box for now. But it's not hard to implement by plugin.

For example, editing the link by a input chip is provided by a plugin here: https://github.com/Saul-Mirone/milkdown/blob/main/packages/preset-commonmark/src/mark/link.ts#L105

And there's also a community plugin called [milkdown-plugin-image-picker](https://github.com/LittleSound/milkdown-plugin-image-picker) that provides a file picker for image.

So I think it would be greate if you're willing to create a community plugin to midify the alt text of an image.

**Reply by leo-petrucci** [reply]
That's exactly what I need, thank you!
